premium/eagle-eye/crowd-eagle-eye/crowd/eagle_eye/sync.py

```diff
@@ -0,0 +1,61 @@
+from crowd.eagle_eye.apis.vector_api import VectorAPI
+from crowd.eagle_eye.apis.embed_api import EmbedAPI
+from crowd.eagle_eye.models import Vector, Payload
+import pinecone
+import os
+
+import dotenv
+found = dotenv.find_dotenv(".env.sync")
+dotenv.load_dotenv(found)
+
+
+pinecone.init(api_key=os.environ.get("PINECONE_API_KEY"), environment="us-east-1-aws")
+index = pinecone.Index("crowddev-prod")
+filters = [
+    {"platform": {"$in": ["hacker_news"]}, "timestamp": {"$gt": 1666681782}},
+    {"platform": {"$in": ["devto"]}, "timestamp": {"$gt": 1666681782}}
```

Comment:
Let's either put the 1970 timestamp in a variable with a descriptive name or comment the file to clarify that this is an old date

Reply:
Sorry, I forgot to mention that this file will be deleted